File: JavaScript Solutions/1035.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split(' ');

let A = parseInt(lines.shift()),
    B = parseInt(lines.shift()),
    C = parseInt(lines.shift()),
    D = parseInt(lines.shift());
    
const 
    CD = C+D, 
    AB = A+B;
    
if ( (B>C) && (D>A) && (CD>AB) && (C > 0) && (D > 0) &&  (A%2===0) )
    console.log("Valores aceitos");
else
    console.log("Valores nao aceitos");

File: Python Solutions/1035.py
dados = input().split()

A= int(dados[0])
B= int(dados[1])
C= int(dados[2])
D= int(dados[3])

CD= C+D
AB= A+B

if ((B>C) and (D>A) and (CD>AB) and (C>0) and (D>0) and (A%2==0)):
    print("Valores aceitos")
else:
    print("Valores nao aceitos")
